frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx

```diff
@@ -8,12 +11,14 @@ export type DestinationsFiltersProps = {
     hideSearch?: boolean
     hideShowPaused?: boolean
     hideKind?: boolean
+    askForFeedback?: boolean
 }
 
 export function DestinationsFilters({
     hideSearch,
     hideShowPaused,
     hideKind,
+    askForFeedback = true,
```

Comment:
Everything else is "hideFoo" - generally speaking when making a change like this it is best to stick with the same naming style please
```suggestion
    hideFeedback,
```